packages/execution-context/src/lib/contextStore.ts

```diff
@@ -0,0 +1,79 @@
+import { AsyncLocalStorage } from "node:async_hooks";
+
+import { type ExecutionContext } from "../types/types";
+
+globalThis.threadLocalStorage ||= new AsyncLocalStorage();
+
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
```

Comment:
Why does this class exist if all of its members are static? Can we just export functions?

Reply:
We can, yes. I'll refactor it to export only functions.